src/Microsoft.ML.StaticPipe/TextStaticExtensions.cs

```diff
@@ -585,13 +585,13 @@ public bool Equals(Reconciler other)
         /// Text representation of original values are stored in the slot names of the  metadata for the new column.Hashing, as such, can map many initial values to one.
         /// <paramref name="invertHash"/> specifies the upper bound of the number of distinct input values mapping to a hash that should be retained.
         /// <value>0</value> does not retain any input values. <value>-1</value> retains all input values mapping to each hash.</param>
-        public static Vector<float> ToNgramsHash(this VarVector<Key<uint, string>> input,
-            int hashBits = 16,
+        public static Vector<float> ApplyNgramHashing(this VarVector<Key<uint, string>> input,
```

Comment:
>ApplyNgramHashing [](start = 36, length = 17)

in dynamic it's `Produce`

Reply:
Sure.

---
In reply to: [263980246](https://github.com/dotnet/machinelearning/pull/2898#discussion_r263980246) [](ancestors = 263980246)